components/AudioPlayer/AudioPlayer.js

```diff
@@ -0,0 +1,138 @@
+import {
+  Clear,
+  Pause,
+  PlayArrowRounded,
+  VolumeDown,
+  VolumeOff,
+  VolumeUp,
+} from '@mui/icons-material';
+import { convertTime } from '../../utils';
+import { Slider } from '@mui/material';
+import React from 'react';
+import { useLessonsProgress } from '../../hooks';
+import styles from './AudioPlayer.module.css';
+import { useRouter } from 'next/router';
+import cx from 'classnames';
+
+export function AudioPlayer({ openAudio }) {
+  const [pause, setPause] = React.useState(true);
+  const { lessonsProgress, updateLessonProgress } = useLessonsProgress();
+  const [position, setPosition] = React.useState(0);
+  const [progressPosition, setProgressPosition] = React.useState(0);
+  const [seekActivated, setSeekActivated] = React.useState(false);
+
+  const audioRef = React.useRef();
+  const router = useRouter();
+  React.useEffect(() => {
+    if (pause) {
+      audioRef.current?.pause();
+    } else {
+      audioRef.current?.play();
+    }
+  }, [pause]);
+  React.useEffect(() => {
+    if (seekActivated) {
+      audioRef.current.currentTime = progressPosition;
+      setSeekActivated(false);
+    }
+  }, [progressPosition, seekActivated]);
+
+  const updateProgress = (currentTime) => {
+    setProgressPosition(currentTime);
+    updateLessonProgress(openAudio.id, currentTime / openAudio.duration);
+  };
+
+  return (
+    <div
+      className={styles.expandedContent}
+      onKeyDown={(e) => {
+        if (e.keyCode === 27) {
+          router.push('/lessons', undefined, { shallow: true });
+        }
+      }}
+    >
+      <div className={styles.expandedHeader}>
+        <p className={styles.expandedTitle}>LESSON</p>
+
+        <Clear
+          className={styles.expandedClearIcon}
+          onClick={() => router.push('/lessons', undefined, { shallow: true })}
+        />
+      </div>
+      <div className={styles.expandedBody}>
+        <p className={styles.expandedSongName}>{openAudio.title}</p>
+        <p className={styles.expandedSonAuthor}>{openAudio.instructor}</p>
+      </div>
+      <div className={styles.playIconContainer}>
```

Comment:
This should be a button and the onClick should be on it. 99% of the time it's better for buttons to have onClicks for accessibilty.